specification/resources/resource-manager/Microsoft.Authorization/stable/2023-04-01/policySetDefinitionVersions.json

```diff
@@ -0,0 +1,915 @@
+    },
+    "PolicyDefinitionReference": {
+      "type": "object",
+      "properties": {
+        "policyDefinitionId": {
+          "type": "string",
+          "description": "The ID of the policy definition or policy set definition."
+        },
+        "parameters": {
+          "description": "The parameter values for the referenced policy rule. The keys are the parameter names.",
+          "$ref": "./policyAssignments.json#/definitions/ParameterValues"
+        },
+        "policyDefinitionReferenceId": {
+          "type": "string",
+          "description": "A unique id (within the policy set definition) for this policy definition reference."
+        },
+        "groupNames": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The name of the groups that this policy definition reference belongs to."
+        }
+      },
+      "required": [
+        "policyDefinitionId"
+      ],
+      "description": "The policy definition reference."
+    },
+    "PolicyDefinitionGroup": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the group."
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The group's display name."
+        },
+        "category": {
+          "type": "string",
+          "description": "The group's category."
+        },
+        "description": {
+          "type": "string",
+          "description": "The group's description."
+        },
+        "additionalMetadataId": {
+          "type": "string",
+          "description": "A resource ID of a resource that contains additional metadata about the group."
+        }
+      },
+      "required": [
+        "name"
+      ],
+      "description": "The policy definition group."
+    },
+    "PolicySetDefinitionVersion": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PolicySetDefinitionVersionProperties",
+          "description": "The policy set definition version properties."
+        },
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The ID of the policy set definition version."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the policy set definition version."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The type of the resource (Microsoft.Authorization/policySetDefinitions/versions)."
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "readOnly": true,
+          "description": "The system metadata relating to this resource."
+        }
```

Comment:
You can replace all of this with a $ref to the Resource definition in v5 common types.